### ISSUE TITLE
text techniques should not be in 1.4.11 non-text contrast

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#781]
There is now a 'Text in or over graphics' subsection in the techniques for 1.4.11.

The following are text contrast requirements. They do not belong in Non-Text Contrast.
- G18: Ensuring that a contrast ratio of at least 4.5:1 exists between text (and images of text) and background behind the text
- G145: Ensuring that a contrast ratio of at least 3:1 exists between text (and images of text) and background behind the text

A separate issue:  I  view the structure of these techniques grouped under the H4 headings to be a departure from other Understanding docs.
Following existing conventions, ethese should be given Situations (A, B, C, etc) and organized that way. See 1.1.1 for an example.

**Comment by alastc** [comment]
Hi Michael, I added those as the text within images would need to contrast in some circumstances.

Without that a graphic that relies on the embedded text would not pass. 

I suspect you're right about the structure, it was such a long doc I'd run out of energy by the time I got to the end!

**Comment by patrickhlauke** [comment]
would developers (or normal human beings) make the fine distinction between "images of text" or "text within images"?

**Comment by mbgower** [comment]
Closing this issue, as the techniques have been removed. Opening the side issue on the structure of the Understanding page as a separate issue.
